lib/power_enum/schema/schema_statements.rb

```diff
@@ -92,7 +92,7 @@ def create_enum(enum_name, options = {}, &block)
       desc_limit = options[:desc_limit]
       table_options = options[:table_options] || {}
 
-      create_table enum_table_name, table_options do |t|
+      create_table enum_table_name, **table_options do |t|
```

Comment:
I think this forces Rails 6 as the minimum version. Not necessarily a bad thing, but food for thought.